pkg/commands/start_host_windows.go

```diff
@@ -27,42 +29,60 @@ func StartVsockHost(ipv6 bool, hosts map[string]string, upstreamServers []string
 	if err != nil {
 		return err
 	}
-	streamListener, err := vmsock.Listen(vmGUID, vmsock.HostTCPListenPort)
-	if err != nil {
-		return err
+
+	tcpOpts := &rddns.ServerOptions{
+		IPv6:            ipv6,
+		StaticHosts:     hosts,
+		UpstreamServers: upstreamServers,
+		TruncateReply:   false,
 	}
-	streamSrv, err := dns.StartWithListener(
-		&dns.ServerOptions{
-			IPv6:            ipv6,
-			StaticHosts:     hosts,
-			UpstreamServers: upstreamServers,
-			Listener:        streamListener,
-			TruncateReply:   false,
-		})
+	tcpServer, err := startDNSServer(vmGUID, vmsock.HostTCPListenPort, tcpOpts)
 	if err != nil {
-		return err
+		log.Panicf("StartVsockHost failed starting a TCP server: %v", err)
```

Comment:
Hmm, having a more user-understandable error message may be better. `failed to start TCP DNS server: %s`, maybe?